src/login_page/login_page.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Login - Freelancers-Meetpoint</title>
+    <!-- favicon in tab -->
+    <link href="../global/assets/favicon.png" rel="icon">
+    <link href="../global/style.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+  </head>
+  
+  <body>
+    <!-- Header -->
+    <header>
+      <a href="../index.html">
+	<picture alt="logo">
+	  <source media="(max-width:600px)" srcset="../global/assets/logo_with_name_initials.svg">
+	  <source media="(min-width:601px)" srcset="../global/assets/logo_with_name.svg">
+	  <img src="../global/assets/logo_with_name_initials.svg" height="45px" id="header-logo">
+	</picture>
+      </a>
+    </header>
+    
+    <!-- Content -->
+    <main>
+      <form id="login-form" action="TODO" method="post">
+        <ul>
+        <li>
+          <input type="radio" id="login-account-type-employer" name="account-type">
+          <label for="login-account-type-employer">Find Work</label>
+	  <input type="radio" id="login-account-type-freelancer" name="account-type">
+	  <label for="login-account-type-freelancer">Find Talent</label>
+        </li>
+
+        <li>
+          <!-- Username -->
+	  <label for="username">Username</label>
+	  <input type="text" id="username" name="account-name" maxlength="100">
```

Comment:
Change the id to login-account-username for consistency (also update the for attribute in label)
Do same for password -> login-account-password